### PR TITLE
Feature: docker multi arch support

### PR DESCRIPTION
**Pull request by machisuji**
Adds dockerization for arm64/v8 and ppc64le on top of amd64. Doesn't push right now. Once this has been approved via review we can adjust the workflow to actually push.

Todo once approved and **BEFORE** merging:

- [x] Remove branch trigger for this feature branch
- [x] Enable push in last step of docker workflow
- [x] change docker project from community-test to community

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -1,40 +1,75 @@
...
+        uses: docker/setup-buildx-action@v1
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
         with:
-          name: openproject/community
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          tag_semver: true
-          cache: ${{ github.event_name != 'schedule' }}
-      - name: Test
```
Comment (oliverguenther) [inline .github/workflows/docker.yml:35]: You removed the test step from the steps, can we enable this for all builds now?

Comment (oliverguenther) [inline .github/workflows/docker.yml:35]: You removed the test step from the steps, can we enable this for all builds now?

**Review by oliverguenther — CHANGES_REQUESTED** [review]

File: .github/workflows/docker.yml (hunk 1 above)
Comment (machisuji) [reply .github/workflows/docker.yml:35]: Good point. I found a way to test things BEFORE pushing the image, too. Will amend that!

**Comment by machisuji** [comment]
That's now working. The job builds the docker image for amd64, ppc64le and arm64/v8 and pushes everything under one tag:

image

Since this is a feature branch, there was no version detected and the tag is `sha-...`. For the `dev` branch the tag will be `dev` and for actual releases it will be the respective version(s) (major, major.minor, major.minor.patch).

**Comment by machisuji** [comment]
@oliverguenther This is good to merge now IMHO.

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**